data/example/solution_explorer.py

```diff
@@ -1,10 +1,12 @@
 """A utility script to visualize a solution to the scheduling problem."""
 
+# TODO fix
```

Comment:
The whole file doesn't work with the new structure yet. In my dream scenario this would be similar to the `processing.py` in the benchmark.

Reply:
Ok, that's fine. It's just that a "TODO: fix" is not really helpful for me as a reviewer, because I would need to clone it and look at pylint (or even run and look at the runtime errors) to find out what is actually broken.